### PR TITLE
Created GH discussion link & Bug report config

### PR DESCRIPTION
**Pull request by Sophyia7**
I noticed there was no configuration for reporting bug so I created one and also added feature request for GH discussions.

### CODE REVIEW

**Comment by Sophyia7** [comment]
@mason-at-pieces do check this out

**Comment by mason-at-pieces** [comment]
So when there is a new release for our obsidian plugin, the files in this repo are updated automatically.

@caleb-at-pieces do you know if a file added, would an update just remove the file or only update the existing files?

**Review by caleb-at-pieces — CHANGES_REQUESTED** [review]
I think we should be pushing people towards the support repo instead of letting them create bug reports on this repo.

So just like how we are sending them to discussions, we should probably send them to https://github.com/pieces-app/support/issues/new?template=bug.yml for the bug report

**Comment by caleb-at-pieces** [comment]
> So when there is a new release for our obsidian plugin, the files in this repo are updated automatically.
> 
> @caleb-at-pieces do you know if a file added, would an update just remove the file or only update the existing files?

The CICD will not affect these templates

**Comment by mason-at-pieces** [comment]
> I think we should be pushing people towards the support repo instead of letting them create bug reports on this repo.
> 
> So just like how we are sending them to discussions, we should probably send them to https://github.com/pieces-app/support/issues/new?template=bug.yml for the bug report

Yep I agree, @Sophyia7 can you remove the bug template and instead just add another link under the feature request one that goes to the issues page on our support repo?

**Comment by Sophyia7** [comment]
@mason-at-pieces @caleb-at-pieces  updated it. Lmk what you think.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Submit Your Bug
```
Comment (caleb-at-pieces) [inline .github/ISSUE_TEMPLATE/bug.yml:3]: 'Report a Bug'

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    url: https://github.com/pieces-app/support/issues/new?template=bug.yml
+    about: Submit your bug or issues here for the Pieces team to resolve!
```
Comment (caleb-at-pieces) [inline .github/ISSUE_TEMPLATE/bug.yml:5]: 'Report a bug to the Pieces team.'

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Feature or Enhancement Request
```
Comment (caleb-at-pieces) [inline .github/ISSUE_TEMPLATE/config.yml:3]: 'Request a Feature'

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+    url: https://github.com/pieces-app/support/discussions
+    about: Post your idea for obsidian in discussions for the Pieces team and community to discuss!
```
Comment (caleb-at-pieces) [inline .github/ISSUE_TEMPLATE/config.yml:5]: 'Post your idea for the Pieces Obsidian plugin in discussions for the Pieces team and community to discuss!'

**Review by caleb-at-pieces — CHANGES_REQUESTED** [review]
some small copy adjustments and we are GTG

**Comment by mason-at-pieces** [comment]
Can we make sure we get these copy updates on all other repos as well @Sophyia7?

**Pull request closed without merging**